app/pulp/app/response.py

```diff
@@ -0,0 +1,27 @@
+from gettext import gettext as _
+
+from rest_framework.response import Response
+
+
+class OperationPostponedResponse(Response):
+    """
+    An HTTP response class for returning 202 and a list of spawned tasks.
+    """
+
+    def __init__(self, task_results, task_group_id=None):
+        """
+        :param task_results: List of AsyncResult objects that are used to generate the response.
+        :type task_group_id: List of AsyncResult
+        :param task_group_id: Task group id, if any, for all the task results
+        :type task_gropu_id: str
+        """
+        if not isinstance(task_results, list):
+            raise RuntimeError(_('task_results must be an instance of list.'))
```

Comment:
``TypeError`` would be an appropriate exception to use, and has the advantage of being more specific.